samples/Ring/weblib.ring

```diff
@@ -0,0 +1,3039 @@
+Import System.Web
```

Comment:
Yes, this file is still too large and contains a lot of data that, whilst relevant to a real-world use, is not relevant to the structure of the language. For example the `htmlcssattributes` is waaaaaaay bigger than is really needed: 65 lines for a single variable 😱!!

There's also a fair amount of "duplication" in the huge number of similar functions.

My naïve "I know nothing about ring" point of view believes this can be cut down quite considerably, or replaced with a smaller, more representative, example.